vision/test/visionTest.php

```diff
@@ -169,14 +165,14 @@ public function testLogoCommandGcs()
         $bucketName = $this->requireEnv('GOOGLE_STORAGE_BUCKET');
 
         $path = 'gs://' . $bucketName . '/vision/logo.jpg';
-        $output = $this->runCommand('logo', ['path' => $path]);
+        $output = $this->runFunctionSnippet('detect_logo_gcs', ['path' => $path]);
         $this->assertStringContainsString('Google', $output);
     }
 
     public function testLocalizeObjectCommand()
```

Comment:
You might want to remove the term `localize` from test method name? 
Something like:
```suggestion
    public function testDetectObjectCommand()
```

Reply:
You're right - I think the sample was renamed (if you look at the region tag in `detect_object.php`, it's still using `localized`). 

I agree that "detect" is preferred. I'll update line 183 as well!